models/schema_site_service_graph.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+type SiteServiceGraph struct {
+	Ops   string                 `json:",omitempty"`
+	Path  string                 `json:",omitempty"`
+	Value map[string]interface{} `json:",omitempty"`
+}
+
+func NewSchemaSiteServiceGraph(ops, path string, siteServiceGraphList []interface{}) *PatchPayloadList {
```

Comment:
The code is good, but we are not doing any logic in here so we could create PatchPayloadList directly in the provider code. How should we deal with these situations? Or shall we always provide a wrapped function for consistency purposes?